similarity/config/datastore.properties

```diff
@@ -0,0 +1,3 @@
+ENDPOINT	http://smaug.cs.illinois.edu:8080
```

Comment:
without the access key. I got access denied. See error message below:

error occured
ErrorResponse(code=AccessDenied, message=Access denied, bucketName=readonly.org.cogcomp.wordembedding, objectName=1.5/phrase2vec.txt, resource=/readonly.org.cogcomp.wordembedding/1.5/phrase2vec.txt, requestId=14CA64275606BDE5, hostId=null)
request={method=HEAD, url=http://smaug.cs.illinois.edu:8080/readonly.org.cogcomp.wordembedding/1.5/phrase2vec.txt, headers=Host: smaug.cs.illinois.edu:8080
User-Agent: Minio (x86_64; x86_64) minio-java/dev
x-amz-date: 20170622T082546Z
}